ui_components/county_selector.R

```diff
@@ -0,0 +1,8 @@
+county_selector <- function(county_list) {
```

Comment:
Same comment as above on the UI - over-abstracted for early development and fast iteration.